### DISCUSSION TITLE
Help! Time dimesion in WebMapServiceCatalogItem

### DISCUSSION THREAD

**Discussion by rnnchy** [TerriaJS/terriajs#7286] [Q&A] [answered]
Hi Folks! I want to change the time dimesion to be interval instead of timestamp, how to do that in WebMapServiceCatalogItem? I want to achieve something like this: time=2016-01-01T00:00:00Z/2017-01-01T00:00:00Z

```
service=WMS
version=1.3.0
request=GetMap
layers=geoagri:iroadmap_machinery_with_date
styles=
transparent=true
format=image/png
exceptions=XML&
time=2016-01-01T00:00:00Z/2017-01-01T00:00:00Z
bbox=15028131.257091936,1252344.271424327,15654303.392804097,1878516.4071364924
width=256
height=256
crs=EPSG:3857
```

**Comment by zoran995** [comment]
Hi, to make sure I understand correctly what you are asking you are trying to limit time availability of the layer to be smaller subset to what is returned by get capabilities and to render only those times as available in terria? 

i.e. adding url https://nationalmap.gov.au/proxy/_1d/https://nationalmap.gov.au/proxy/_0d/https://gis.aremi.data61.io/bom/wms?transparent=true&format=image%2Fpng&exceptions=XML&styles=solar_satellite&version=1.3.0&time=PT16H%2F2014-01-01T23%3A00%3A00Z&service=WMS&request=GetCapabilities&layers=Solar_Satellite_DNI_2014&bbox=10018754.17139462%2C-10018754.17139462%2C20037508.342789244%2C0&width=256&height=256&crs=EPSG%3A3857
should show only 16h of data as available for the url instead of entire year 2014 as advertised by getCapabilities? I am not sure this is supported but can check more in details later

**Reply by rnnchy** [reply]
I want to get all the data from 2016-2017, 2016-2018, and 2016-2019, making it cumulative instead of just obtaining data from a single year.

**Reply by zoran995** [reply, marked as answer]
Hopefully, I got it correct this time, you would like terria timeline to have start-end range. I don't think it is supported currently, AFAIK timeline supports getting only a single point in time. I was pretty sure that someone asked for something similar but can't find it right now
